lua/nvim-tree.lua

```diff
@@ -145,6 +145,7 @@ local keypress_funcs = {
         end
       end
     )
+    table.remove(process.args)
```

Comment:
i think the process table being used inside the callback is some kind of memory leak maybe, at least i think the GC is not properly cleaning up because of this. Thanks for the fixes though, i do not use this feature myself so couldn't see the issues :)